src/scenes/NoOrgRouter/index.tsx

```diff
@@ -1,34 +1,20 @@
 import React from 'react';
 import { Navigate, Route, Routes } from 'react-router-dom';
 
-import { makeStyles } from '@mui/styles';
-
 import NoOrgLandingPage from 'src/components/emptyStatePages/NoOrgLandingPage';
 import { APP_PATHS } from 'src/constants';
 import { useUser } from 'src/providers';
 import MyAccountRouter from 'src/scenes/MyAccountRouter';
 import OptInFeaturesView from 'src/scenes/OptInFeatures';
 import useEnvironment from 'src/utils/useEnvironment';
 
-const useStyles = makeStyles(() => ({
-  container: {
-    '& .MuiAppBar-root': {
-      width: '100%',
-    },
-  },
-}));
-
 const NoOrgRouter = () => {
   const { isProduction } = useEnvironment();
   const { reloadUserPreferences: reloadPreferences } = useUser();
-  const classes = useStyles();
 
   return (
     <Routes>
-      <Route
-        path={`${APP_PATHS.MY_ACCOUNT}/*`}
-        element={<MyAccountRouter className={classes.container} hasNav={false} />}
-      />
+      <Route path={`${APP_PATHS.MY_ACCOUNT}/*`} element={<MyAccountRouter hasNav={false} />} />
```

Comment:
Don't we need the container class here?

Reply:
Good catch! 🙌 💯

So for this one I wasn't sure how I was going to apply the styles to the `Route` component since it doesn't support `sx` props and we have no control over this component as it comes from `react-router-dom`...

When I looked into updating the `MyAccountRouter` component, I discovered the `className` prop is passed down through several layers of components (`MyAccountRouter` > `MyAccountView` > `MyAccountContent`), but is ultimately never used.

Reply:
Everything looks good in the views under that router, perhaps it was not actually being used.